code/game/atoms.dm

```diff
@@ -139,7 +139,7 @@
 /atom/proc/examine(mob/user, var/distance = -1, var/infix = "", var/suffix = "")
 	//This reformat names to get a/an properly working on item descriptions when they are bloody
 	var/f_name = "\a [src][infix]."
-	if(src.blood_DNA && !istype(src, /obj/effect/decal))
+	if(src.blood_DNA && !istype(src, /obj/effect/decal) && !istype(src, /obj/effect/rune))
```

Comment:
`istype(src, ...)` is something we should never do, you'll need another solution here

Reply:
Do you need me to fix the !istype(src, /obj/effect/decal) as well?  There's a lot of instances of istype(src, ...) in the code.  Do you want me to avoid messing with the examine proc, or just find an alternate solution?  Do you have a recommendation as to what I should use?

Curious, how come it's a bad thing to do?  Does it do bad things to the server load?

Reply:
Nevermind!  Myazaki told me what's up, that I should override the rune's own examine proc.  I'll push it later, when I've got some sleep and work done!

Reply:
Fixed.  Did I do it right?

Reply:
I'll mark this as resolved?  I'm not sure how this works.